compiler-core/src/parse.rs

```diff
@@ -209,7 +209,7 @@ pub struct Parser<T: Iterator<Item = LexResult>> {
     tok0: Option<Spanned>,
     tok1: Option<Spanned>,
     extra: ModuleExtra,
-    doc_comments: VecDeque<(u32, String)>,
+    doc_comments: VecDeque<(u32, u32, String)>,
```

Comment:
It's not clear to me what these values are. Could you make a struct for them so that they have names. Thank you